frontend/vendor/js/diff.min.js

```diff
@@ -0,0 +1,307 @@
+/*!
+
+ diff v3.1.0
```

Comment:
`npm install diff --save` adds the package to the `package.json` file and installs it in `node_modules`. But `lineman build` doesn't include them in `frontend/generated/js/app.js` (I tried it). I think we would need a module loader like[ browserify](http://browserify.org/) or [requirejs](http://requirejs.org/) in order to do that, or maybe `lineman` can do it and I'm missing something?